index.md

```diff
@@ -16,49 +16,33 @@ order: 1
 ## Introduction
 {:#introduction}
 
-Web accessibility evaluation tools are software programs or online services. They help determine if a Web site is accessible. This document highlights features of evaluation tools. It discusses different aspects of evaluation tools to consider when selecting these tools. 
-
```

Comment:
I agree with the comments made by @bakkenb that the paragraphs below are repetitive and not needed.